config/.eslintrc.js

```diff
@@ -32,6 +32,10 @@ module.exports = {
     ],
     rules: {
         "@typescript-eslint/explicit-function-return-type": "off",
```

Comment:
Personally, I absolutely can't stand reading code that doesn't have return types in function signatures. It's vital information, on par with input types. It forces you to have to read through all of the code just to figure out what it's supposed to return. Sure if you have intellisense available it can tell you the inferred type, but that doesn't tell you what the _intended_ return type is. Within a function, explicit types matter less since it's just internal implementation details (e.g. anonymous callback with inferred types is fine). But a class method or exported function defines a contract with other parts of the code, so should always have proper typing in my opinion.

Reply:
As discussed we'll keep the boundary-types rule enabled, but leave `explicit-function-return-type` disabled for now as a compromise.